### PR TITLE
LPD-24508 Integration Test for NavigationControlMenuEntry

### PR DESCRIPTION
**Pull request by gergelyszalay**
Hi,

I am resending the test from: https://github.com/brianchandotcom/liferay-portal/pull/150034

Thank you!

### CODE REVIEW

**Comment by gergelyszalay** [comment]
ci:test:sf

**Comment by gergelyszalay** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/product/navigation/control/menu/test/EditFileEntryHeaderProductNavigationControlMenuEntryTest.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+		if (fileEntry != null) {
+			mockHttpServletRequest.setAttribute(
+				WebKeys.DOCUMENT_LIBRARY_FILE_ENTRY, fileEntry);
+		}
+
+		mockHttpServletRequest.setAttribute(
+			WebKeys.THEME_DISPLAY, _getThemeDisplay());
+
+		return mockHttpServletRequest;
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/product/navigation/control/menu/test/EditFileEntryHeaderProductNavigationControlMenuEntryTest.java:166]: You need to set the attributes in alphabetical order: DOCUMENT_LIBRARY_FILE_ENTRY < JAVAX_PORTLET_RESPONSE < THEME_DISPLAY

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/product/navigation/control/menu/test/EditFileEntryHeaderProductNavigationControlMenuEntryTest.java (hunk 1 above)
Comment (gergelyszalay) [reply modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/product/navigation/control/menu/test/EditFileEntryHeaderProductNavigationControlMenuEntryTest.java]: Hi @adolfopa ,
I sent the changes. Thank you!

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**